app/views/articles/show.html.erb

```diff
@@ -31,6 +31,40 @@
     </script>
   <% end %>
 
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "<%= article_url(@article) %>"
+    },
+    "url": "<%= article_url(@article) %>",
+    "image": "<%= article_social_image_url(@article) %>",
+    "publisher": {
+      "@context": "http://schema.org",
+      "@type": "Organization",
+      "name": "<%= community_qualified_name %>",
+      "logo": {
+        "@context": "http://schema.org",
+        "@type": "ImageObject",
+        "url": "<%= cloudinary(SiteConfig.logo_png, 192, "png") %>",
+        "width": "192",
+        "height": "192"
+      }
+    },
+    "headline": "<%= @article.title %>",
+    "author": {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "url": "<%= user_url(@user) %>",
+      "name": "<%= @user.name %>"
+    },
+    "datePublished": "<%= @article.published_timestamp %>",
+    "dateModified": "<%= @article.edited_at || @article.published_timestamp %>"
```

Comment:
If you look at the implementation of `published_timestamp`, we return the ISO date so here we have to do the same, otherwise the date will have a different format, for example:

```ruby
[2] pry(main)> ar.created_at.to_s
=> "2020-04-01 12:16:31 UTC"
[3] pry(main)> ar.created_at.iso8601
=> "2020-04-01T12:16:31Z"
```

I suggest `@article.edited_at&.iso8601 || @article.published_timestamp`

Reply:
Good catch! 👍